analyzers/src/SonarAnalyzer.Utilities/RuleDescriptors/RuleDetail.cs

```diff
@@ -49,10 +49,9 @@ public sealed class RuleDetail
         public string Remediation { get; }
         public string RemediationCost { get; }
         public List<string> Tags { get; }
-        public List<RuleParameter> Parameters { get; } = new();
-        public List<string> CodeFixTitles { get; } = new();
```

Comment:
Removing this `CodeFixTitles` property is the important thing. The rest of this PR removes the complicated computing of this property.